File: C++/1153.cpp
#include <iostream>

using namespace std;

int fat (int n) {
if ((n==1) || (n==0)) return 1;               
   else
      return fat(n-1)*n;
}
int main(){

    int entrada, fatorial = 1;

    cin >> entrada;

    fatorial = fat(entrada);

    cout << fatorial << endl;
    return 0;
}

File: JavaScript/1153.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

function fat(numero){
    if(numero == 0 || numero == 1){
        return 1;
    }else{
        return fat(numero-1) * numero;
    }
}

let entrada = parseInt(lines[0].split(' '))
let fatorial;

fatorial = fat(entrada)

console.log(fatorial);